Tests/MongoSwiftTests/DocumentTests.swift

```diff
@@ -354,9 +354,23 @@ final class DocumentTests: XCTestCase {
     }
 
     // exclude Int64 value on 32-bit platforms
-    static let overwritables: Document = ["double": 2.5, "int32": Int32(32), "int64": Int64.max, "bool": false, "decimal": Decimal128("1.2E+10")]
-
-    static let nonOverwritables: Document = ["string": "hello", "nil": nil, "doc": ["x": 1] as Document, "arr": [1, 2] as [Int], "oid": ObjectId()]
+    static let overwritables: Document = [
+        "double": 2.5,
+        "int32": Int32(32),
+        "int64": Int64.max,
+        "bool": false,
+        "decimal": Decimal128("1.2E+10"),
+        "oid": ObjectId(),
+        "timestamp": Timestamp(timestamp: 1, inc: 2),
+        "datetime": Date(msSinceEpoch: 1000)
```

Comment:
Just confirming that "oid" was moved from `nonOverwritables`, and "timestamp" and "datetime" were added (previously not tested at all).

Reply:
Yup that's exactly right. Previously, we were not testing `"timestamp"` or `"datetime"`, in `nonOverwritables` or `overwritables`, so when I noticed that I made sure to include them now.